Lib/test/test_inspect.py

```diff
@@ -1986,6 +1986,13 @@ def test(a, b:'foo'=10, *args:'bar', spam:'baz', ham=123, **kwargs:int):
                            ('kwargs', ..., int, "var_keyword")),
                           ...))
 
+    def test_signature_without_self(self):
+        class A:
+            f = functools.partialmethod((lambda *args: ...), 1)
```

Comment:
I think it would look clearer if use a local function instead of a lambda.

Are there other tests for partialmethod? If not, it would be nice to add tests for more common partialmethod.

Does this work if there are keyword-only parameters after `*args`? Does this work if a partialmethod is decorated with classmethod or staticmethod?